test/db/cmd/tables

```diff
@@ -115,13 +115,13 @@ imports          loc.imp._ITM_deregisterTMCloneTable
 .----------------------------------------------------------------------------------------------------.
 | space     | size  | name                                                                           |
 )----------------------------------------------------------------------------------------------------(
-| classes   | 1     | class.std::ostream                                                             |
 | classes   | 1     | class.std::ios_base::Init                                                      |
-| classes   | 1     | class.std::basic_ostream_char__std::char_traits_char_____std::operator____std  |
-| classes   | 1     | class.std::basic_ostream_char__std::char_traits_char_____std::endl_char__std   |
+| classes   | 1     | class.std::ostream                                                             |
 | classes   | 1     | class.B                                                                        |
-| classes   | 1     | class.A                                                                        |
+| classes   | 1     | class.std::basic_ostream_char__std::char_traits_char_____std::endl_char__std   |
 | classes   | 1     | class.C                                                                        |
+| classes   | 1     | class.std::basic_ostream_char__std::char_traits_char_____std::operator____std  |
+| classes   | 1     | class.A                                                                        |
```

Comment:
Thanks for your review. IMHO, they are sorted correctly regarding `size`. Since there are many items with `size=1` and the sort algorithm is not stable, I changed the test to sort by name which is unique in next commit.